.buildkite/pipeline.yml

```
@@ -26,8 +26,32 @@ steps:
         environment:
           - PYTHONDONTWRITEBYTECODE=1
 
+  - label: ":docker: :hadolint:"
```

Maybe this could be a bit more descriptive, since hadolint isn't entirely obvious (I know platform currently isn't, but maybe it should be)?

```suggestion
  - label: ":docker: :hadolint: linting Dockerfiles"
```

---

Okay I've pushed the suggested changes 👍